.github/workflows/ci.yml

```diff
@@ -72,14 +248,13 @@ jobs:
           path: dist/
```

Comment:
The if condition for this step (Upload yourloops build) should be changed to `if: github.ref_name == 'dblp'`. We want to publish a latest docker image only when pushing on dblp right ?